underlay.repos

```diff
@@ -0,0 +1,5 @@
+repositories:
+  fog_msgs:
+    type: git
+    url: https://github.com/tiiuae/fog_msgs.git
+    version: 54fd0bd7184c863339b98b0bd2795d9d72608344
```

Comment:
Similar to comment about rosdep.yaml, no need for defining underlay.repos. However, file should exist with the current docker file. So, either the contents of this file can be removed. Or, the Dockerfile can be modified to copy `underlay.repos*` instead of underlay.repos.